proto/provenance/expiration/v1/expiration.proto

```diff
@@ -0,0 +1,35 @@
+// Since: provenance 1.12
+syntax = "proto3";
+package provenance.expiration.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package          = "github.com/provenance-io/provenance/x/expiration/types";
+option java_package        = "io.provenance.expiration.v1";
+option java_multiple_files = true;
+
+// Params defines the set of params for the expiration module.
+message Params {
+  option (gogoproto.equal)            = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  // the deposit amount for holding module assets on chain
+  cosmos.base.v1beta1.Coin deposit = 2 [(gogoproto.nullable) = false];
```

Comment:
Is there a reason to limit this to a single `Coin` instead of using `Coins`?

Reply:
Yes, my understating is that we're only supporting `nhash` deposits. 

@derekadams is the ^^^ assumption correct or should this be a repeated field of type `Coins`?

Reply:
I agree that `Coins` makes more sense 👍

Reply:
Switched to using `Coins`